### PR TITLE
fix: dark mode icons

### LINKED ISSUE #17: Fix ELIXIR logos in light mode
The letters in the ELIXIR logo should be black/dark in light mode, otherwise they are not visible well. This applies to both the active/selected and inactive state.

See screenshot:
elixir_logo_light_dark

### PR DESCRIPTION
**Pull request by tanya17-05**
Closes #17 
previous:
prev

after
later

### CODE REVIEW

**Comment by tanya17-05** [comment]
Done, please review

**Review by uniqueg — DISMISSED** [review]

File: components/Partners.js [hunk 1]
```diff
@@ -22,7 +24,7 @@ function Partners({ partners }) {
           <div className="w-full rounded-lg border-2 shadow-lg hover:shadow-md my-5 hover:bg-gray-100 dark:bg-gray-900 dark:hover:bg-gray-800 dark:border-gray-800 dark:hover:border-gray-900 cursor-pointer ">
             <div className="flex md:flex-row flex-col">
               <img
```
Comment (git-anurag-hub) [inline components/Partners.js:26]: Hey @tanya17-05, can you please add the ID tag to this img tag with value darkMode.

**Review by git-anurag-hub — CHANGES_REQUESTED** [review]

File: components/Partners.js (hunk 1 above)
Comment (tanya17-05) [reply components/Partners.js:26]: Sure, do I need to add id attribute to light mode logos as well??

Comment (tanya17-05) [reply components/Partners.js:26]: done @git-anurag-hub

**Review by uniqueg — APPROVED** [review]

**Pull request merged**